gradle/integTest.gradle

```diff
@@ -31,8 +31,10 @@ ext.integTestTasks = tasks.withType(IntegrationTest)
 
 integTestTasks.all { IntegrationTest task ->
```

Comment:
❌ Should remove all the `crossVersionTest` code from this file and move all *CrossVersionTest* to the new sourceSet.